libs/sdk-ui-kit/src/Header/generateHeaderDataMenuItems.ts

```diff
@@ -0,0 +1,42 @@
+// (C) 2007-2020 GoodData Corporation
+import { IHeaderDataMenuItem } from "./HeaderDataMenu";
+
+/**
+ * @internal
+ */
+export function generateHeaderDataMenuItems(
```

Comment:
I believe this logic has no place in the SDK8, it has no reasonable use outside of our applications. I understand the desire to share code across multiple applications, but this is pure implementation detail of those applications and I wonder if it should be shared in a public repository only because we have no better place to put it at the moment.

I would suggest moving the Header component here (as you did in this PR), but keeping the logic that fills it in the individual apps (or sharing that logic somewhere else/differently).

Cc @lupko

Reply:
yes agreed. this is very app specific; it should be with the apps. even if we have to duplicate the config.

Reply:
At this time, we have to create and implement this function for 3 apps ( gdc-datasource-management, gdc-msf-modeler and gdc-freemium-load) seperately. The code could be duplicated in 3 times. Do you think we still need to keep as current ? 
cc @no23reason @lupko

Reply:
After discussion with team, We won't move the logic to SDK8 due to no suitable place at this time. We will keep the logic code in 3 apps. Thanks